test/common/ipc_common.c

```diff
@@ -131,6 +131,9 @@ int run_consumer(int port, umf_memory_provider_ops_t *provider_ops,
         return -1;
     }
 
+    umf_memory_pool_handle_t pool;
+    umf_result = umfPoolCreate(pool_ops, provider, pool_params, 0, &pool);
```

Comment:
Return if pool not created? Same in the producer.

Reply:
Error handling missing

Reply:
done